meetings/2023-05-10.md

```diff
@@ -0,0 +1,124 @@
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Sarven
+* Pavlik
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+### Overview of today's agenda
+
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* eP: ...
+* SC: Played with blob and something else IIRC in dokieli's "save as.." for media/binary resources.
+* eP: we use authenticated fetch. We get a blob from response. If you pass that to url createObjectURL, the resulting URL is something you can use in source of the image or as the download file. So far it works. Haven't tried with images that exceed available memory. For now we use that method for most basics and whatever attachments people want to use.
+* eP: In sai-js I use UUID.. no intention to use anything human-readable in the URI. Re download, I plan to have it as part of the resource description - part of the metadata not URI. Possibly also include media type information. CSS takes content-type from the request. It preserves the extension if file-based system used. I was thinking of using content-type in the resource description. If you have some kind of attachment, you could get some kind of icon to represented. one of the motivations no tot URI because to use human-readable label without changing the URI.
+* SC: You mean "Description Resource" right?
+* eP: Yes. linked with "describedby"
+* SC: If you don't have human readable label, everything after the last slash is used
+* SC: One convention used when no RDF label is available, then use string after # or last /.
+* JZ: With the way URIs are built, can't do anything - not reliably reveal users' name. For practical reasons need to do something. In solidOs there is a method called label, and it tries to come up with a human-readable label. It is a bit more complicated than # / string but same idea.
+* eP: Shapes trees provide a way which predicate to use for the human.
```

Comment:
```suggestion
* eP: Shapes trees provide a way which predicate to use for the human. [`st:describesInstance`](https://shapetrees.org/TR/specification/#descriptions) (it will move from description directly to a shape tree [shapetrees#77](https://github.com/shapetrees/specification/issues/77))
```